packages/amplication-client/src/Resource/git/AuthResourceWithGithubCallback.tsx

```diff
@@ -2,8 +2,9 @@ import { useEffect } from "react";
 import { gql, useMutation } from "@apollo/client";
 import { useTracking } from "../../util/analytics";
 import { AnalyticsEventNames } from "../../util/analytics-events.types";
+import { EnumGitProvider } from "../../models";
 
-const AuthResourceWithGitCallback = () => {
+const AuthResourceWithGithubCallback = () => {
   const { trackEvent } = useTracking();
   const [completeAuthWithGit] = useMutation<boolean>(CREATE_GIT_ORGANIZATION, {
```

Comment:
again, this is not a code that I added or changed. It's here because I just renamed the file because now we also have Bitbucket callback that has a different flow